code-pushup.config.ts

```diff
@@ -71,6 +81,11 @@ const config: CoreConfig = {
         { type: 'group', plugin: 'eslint', slug: 'suggestions', weight: 1 },
       ],
     },
+    {
+      slug: 'performance',
+      title: 'Performance',
+      refs: [...lighthouseCorePerfGroupRefs],
+    },
```

Comment:
Please put this under _Custom checks_ category, it's intended for these example plugins, so we that we can easily separate meaningful results for the CLI codebase (portal login page has nothing to do with CLI performance). I introduced this category in #399

Reply:
done